frontend/src/ts/utils.ts

```diff
@@ -405,3 +408,15 @@ export const getLocaleLngDescription = (description: {
   const localeLng = getLngFromLocalStorage();
   return description[localeLng];
 };
+
+export const getAbsoluteStartEndRange = () => {
```

Comment:
we already has `moment` library. below is sample code.
```
const endDate = moment(); // 当前日期
const startDate = moment().subtract(7, 'days'); // 7天前的日期
return {
    type: 'absolute',
    startDate: startDate.format('YYYY-MM-DD'),
    endDate: endDate.format('YYYY-MM-DD'),
  }
```

Reply:
done.